fizzbuzz/COBOL/nathaniel_priestley/README.md

```diff
@@ -0,0 +1,22 @@
+TO OBTAIN COBOL COMPILER:
+
+WEBSITE: https://brew.sh/
+1. INSTALL HOMEBREW:
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+2. INSTALL COBOL COMPILER:
+brew install gnu-cobol
+
+3. VERIFY COMMAND:
+cobc -v
+
+
+TO COMPILE:
+cobc -free -x -o FizzBuzz-exe FizzBuzz
```

Comment:
I'd add this compile command to the ./run.sh so that the user only has to run one command instead of two